cql/ManageRareAbnormality.cql

```diff
@@ -752,11 +819,24 @@ define InitialCervicalHistologyInterpretedAsUnspecifiedHsilOrCin2InLast6Years:
       )
   )
 
+define InitialCervicalHistologyInterpretedAsUnspecifiedHsilOrCin2InLast31Years:
```

Comment:
The expression above this - `InitialCervicalHistologyInterpretedAsUnspecifiedHsilOrCin2InLast6Years` - should be removed if it is now unused.

Reply:
Yes, this was changed to 31 years so this expression can be removed.

Reply:
This has been removed in the most recent commit.